webapp/channels/src/actions/emoji_actions.js

```diff
@@ -132,6 +134,35 @@ export function loadCustomEmojisForCustomStatusesByUserIds(userIds) {
     };
 }
 
+export function loadCustomEmojisForRecentCustomStatuses() {
+    return (dispatch, getState) => {
+        const state = getState();
+        const customEmojiEnabled = isCustomEmojiEnabled(state);
+        const customStatusEnabled = isCustomStatusEnabled(state);
+        if (!customEmojiEnabled || !customStatusEnabled) {
+            return {data: false};
+        }
+
+        const recentCustomStatusesValue = get(state, ReduxPreferences.CATEGORY_CUSTOM_STATUS, ReduxPreferences.NAME_RECENT_CUSTOM_STATUSES);
+        if (!recentCustomStatusesValue) {
+            return {data: false};
+        }
+
+        const recentCustomStatuses = JSON.parse(recentCustomStatusesValue);
+        const emojisToLoad = new Set();
+
+        for (const customStatus of recentCustomStatuses) {
+            if (!customStatus || !customStatus.emoji) {
```

Comment:
Simplify:
```suggestion
            if (!customStatus?.emoji) {
```

Reply:
I'm 0/5 between the two versions of this since, while I like `?.` I'm starting to wonder if the mix of random "?!.!.?!." in some of these checks makes readability worse. This case isn't a problem, but I've gotta be honest, I'm not wanting to change this since it'd require waiting for CI to run again for a minor change